### PR TITLE
Initial version of media block

### PR DESCRIPTION
**Pull request by kirdia**

### CODE REVIEW

File: includes/blocks/media_block.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% block media %}
+    {% if ( fields ) %}
+
+        <div class="eager-wrap">
+            <div class="container">
+                {% if ( fields.attachment ) %}
+                    <img src="{{ fields.attachment }}"/>
+                {% endif %}
+            </div>
+
```
Comment (koyan) [inline includes/blocks/media_block.twig]: you have two empty lines (line 3 and 10) . A reason?

**Comment by angtheod** [comment]
The autoload_static and autoload_classmap files? @kirdia

File: includes/blocks/media_block.twig (hunk 1 above)
Comment (kirdia) [reply includes/blocks/media_block.twig]: No reason. Is not the final markup, it's a dummy one. I will remove them.

**Comment by dashaluna** [comment]
Also, can you merge and fix conflicts :)

**Pull request merged**